On some systems time_t is defined as long long. In this case, the
current release does not contain the appropriate getter to read
values of this type from the database.

https://github.com/kismetwireless/kismet/pull/334

diff -ru a/sqlite3_cpp11.cc b/sqlite3_cpp11.cc
--- a/sqlite3_cpp11.cc	2020-12-06 21:24:46.000000000 +0100
+++ b/sqlite3_cpp11.cc	2021-01-04 11:21:17.239367923 +0100
@@ -412,6 +412,11 @@
     }
 
     template<>
+    long long sqlite3_column_as(std::shared_ptr<sqlite3_stmt> stmt, unsigned int column) {
+        return (long long) sqlite3_column_int64(stmt.get(), column);
+    }
+
+    template<>
     unsigned long long sqlite3_column_as(std::shared_ptr<sqlite3_stmt> stmt, unsigned int column) {
         return (unsigned long long) sqlite3_column_int64(stmt.get(), column);
     }
